src/mw/device/bvh_print.cpp

```diff
@@ -0,0 +1,86 @@
+#pragma once
```

Comment:
Is this just a carbon copy of the other host_print implementation? Why?

Reply:
because of the MADRONA_MWGPU_MAX_BLOCKS_PER_SM #define I forgot the exact detail why but it was required for that reason.

Reply:
can fix that with something I do for mw/device/memory.cpp

Reply:
fixed now